homeassistant/components/nfandroidtv/notify.py

```diff
@@ -190,7 +190,8 @@ def load_file(
             if local_path is not None:
                 # Check whether path is whitelisted in configuration.yaml
                 if self.is_allowed_path(local_path):
-                    return open(local_path, "rb")
+                    with open(local_path, "rb") as file:
+                        return file
```

Comment:
We have to check if this (and the other file-related ones) has the side-effect of the file being closed on return. If yes, then we should postpone this rule, because this means that the structure of the functions has to be changed.